src/site/stages/build/process-cms-exports/index.js

```diff
@@ -1,10 +1,40 @@
 const chalk = require('chalk');
+const get = require('lodash/get');
 
 const { getFilteredEntity } = require('./filters');
 const { transformEntity } = require('./transform');
 const { typeProperties, toId, readEntity } = require('./helpers');
 
-const validateEntity = require('./schema-validation');
+const {
+  validateRawEntity,
+  validateTransformedEntity,
+} = require('./schema-validation');
+
+/**
+ * A list of properties to ignore.
+ *
+ * This list comes from the typeProperties, which we never want to
+ * expand, and a temporary list of properties we don't want to filter
+ * out on a per-content-model basis.
+ *
+ * Additionally, this is useful for temporarily ignoring entity
+ * expansion of certain properties before we've created a filter for
+ * that content model.
+ */
+const ignoreList = typeProperties.concat([
```

Comment:
So is the idea that we will remove things from this as we go?

Reply:
Yeah, exactly. There may end up being things we keep in there, but for the time being, this is like a temporary list until we write the filters for the individual content types which cause the circular references and whatnot.